### PR TITLE
Changing pin to layer

### PR DESCRIPTION
**Pull request by coleman415**
Pin Protocol rebranded to Layer Protocol and requested the logo and links be changed.

### CODE REVIEW

File: config/partner_details.py [hunk 1]
```diff
@@ -63,8 +63,8 @@
      "desc":lazy_gettext("HireGo is building the world's first decentralized peer-to-peer car rental platform using Origin Protocol.")
     },
     {
-    "name":"pinprotocol",
-    "url":"https://pinprotocol.io/",
+    "name":"layerprotocol",
+    "url":"https://layerprotocol.com/",
      "desc":lazy_gettext("The decentralized reputation, incentive, and payments protocol for the global sharing economy, built on the blockchain.")
```
Comment (micahalcorn) [inline config/partner_details.py]: @coleman415 should this be "Layer is the..." or "Layer Protocol is the..."? Their logo doesn't have their name in it.

**Comment by coleman415** [comment]
@micahalcorn changed the description per your note

**Comment by coleman415** [comment]
added nucypher

**Comment by micahalcorn** [comment]
Re-approved ✅

**Pull request merged**